### ISSUE TITLE
NTR: Pyrroline-5-carboxylate dehydrogenase activity low in leukocytes or skin fibroblasts (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8290]
New term request Pyrroline-5-carboxylate dehydrogenase activity low in leukocytes or skin fibroblasts (elthat_111112204422): P5C; WBC

**Comment by pnrobinson** [comment]
@MickeySegal
does this refer to 
[pyrroline-5-carboxylate reductase 1](https://www.ncbi.nlm.nih.gov/gene/5831) and similar enzymes, i.e., related to cutis laxa?

**Comment by MickeySegal** [comment]
https://omim.org/clinicalSynopsis/239510 lists this with a delta 1: https://omim.org/clinicalSynopsis/239510 "Delta-1-pyrroline-5-carboxylate dehydrogenase activity decreased in fibroblasts or leukocytes".

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000695
- label: Reduced tissue delta-1-pyrroline-5-carboxylate dehydrogenase activity
- parents: [Abnormal tissue enzyme activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: PMID:2624476
- def: Concentration or activity of delta-1-pyrroline-5-carboxylate dehydrogenase (P5CDH; EC 1.5.1.12) below the lower limit of normal. P5CDH can be measured in multiple tissues including leukocytes and cultured fibroblasts. P5CDH is a mitochondrial matrix NAD(+)-dependent dehydrogenase which catalyzes the second step of the proline degradation pathway, converting pyrroline-5-carboxylate to glutamate.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8290
- synonyms:
